lib/events.js

```diff
@@ -351,7 +351,7 @@ EventEmitter.prototype.prependOnceListener =
       return this;
     };
 
-// emits a 'removeListener' event iff the listener was removed
+// emits -> Emits a 'removeListener' event if and only if the listener was removed.
```

Comment:
Sorry! Get rid of `emits ->`.

When I typed this:

> `emits` -> `Emits`

...the `->` was meant as an arrow and it meant "Change the thing on the left to be the thing on the right instead". Sorry about the confusion.